pootle/core/initdb.py

```diff
@@ -268,7 +274,7 @@ def create_default_projects():
                  'guide</a>.</div>'),
         'virtual_path': "announcements/projects/"+tutorial.code,
     }
-    ann = Announcement(**criteria)
+    ann = StaticPage(**criteria)
```

Comment:
When I first imported `StaticPage` was for using it on announcement creation, so I imported it as `Announcement`. Now that we are creating regular static pages it feels cleaner to use instead `StaticPage` since using `Announcement` can be confusing.